src/github.com/getlantern/bytecounting/bytecounting_conn.go

```diff
@@ -0,0 +1,58 @@
+// package bytecounting provides mechanisms for counting bytes read/written on
+// net.Conn and net.Listener.
```

Comment:
nit: isn't it really just for counting bytes on a net.Conn, but you can use it from a net.Listener to count bytes on an accepted net.Conn?